docs/IRISMapCubeSequence.rst

```diff
@@ -0,0 +1,103 @@
+===================
+IRISMapCubeSequence
+===================
+
+This class represents Slit Jaw Images from IRIS described by a single World Coordinate
+System (WCS). This class is a list of IRISMapCube objects, one per time step, from the
```

Comment:
This is not correct.  Each sub-cube is 2 or 3D described by a single WCS.  That can be a cube from a single file, or each exposure, for example if the user has used ```explode_along_axis```.

Reply:
Modified